packages/sshnoports/bundles/shell/install.sh

```diff
@@ -164,8 +183,12 @@ post_headless_message() {
 
 install_headless_job() {
   job_name=$1
-  dest="$user_bin_dir/$job_name"
-  cp "$script_dir/headless/$job_name" "$dest"
+  mkdir -p "$user_bin_dir"
+  mkdir -p "$user_log_dir"
+  dest="$user_bin_dir/$job_name.sh"
+  if ! [ -f "$dest" ]; then
+    cp "$script_dir/headless/$job_name.sh" "$dest"
+  fi
```

Comment:
Make necessary folders, and don't overwrite existing job scripts when reinstalling